src/io/flutter/FlutterInitialConfigurator.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter;
+
+import com.intellij.icons.AllIcons;
+import com.intellij.ide.util.PropertiesComponent;
+import com.intellij.openapi.actionSystem.ActionManager;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.fileTypes.FileTypeManager;
+import com.intellij.util.messages.MessageBus;
+import io.flutter.actions.FlutterImportProjectAction;
+
+public class FlutterInitialConfigurator {
+  @SuppressWarnings("UnusedParameters")
+  public FlutterInitialConfigurator(final MessageBus bus,
+                                    final PropertiesComponent propertiesComponent,
+                                    final FileTypeManager fileTypeManager) {
+    // Intercept and redirect Import actions.
+    if (!FlutterUtils.isAndroidStudio()) {
```

Comment:
TODO(messick) Ensure something is done in AS to handle project import properly.
Either remove the guard clause (that I asked @pq to add) if there is no difference in the AS initialization code, or add equivalent logic to the Flutter initializer.